### PR TITLE
Minor UI improvements to rubric settings modal

### LINKED ISSUE #9458: Give points column in rubric edit view a minimum width
If descriptions are "too long" the points column is not wide enough for the value to be visible:

image

### PR DESCRIPTION
**Pull request by jonatanschroeder**
* Sets a minimum width to the number of points (handles case where explanation and grader note are too wide), resolves #9458;
* Increases the width of the rubric modal;
* Fixes the view of the "visible to students" radio (would overlap radio with label);
* Fixes colspan of message when no items are set;
* ~~Makes the rubric settings modal draggable (#9359)~~. Removed because of #9465.

### CODE REVIEW

**Comment by jonatanschroeder** [comment]
> Something in this PR broke the `(i)` tooltips in the modal. They work on `master` but not on this branch.

This is the issue: https://stackoverflow.com/questions/13731400/jqueryui-tooltips-are-competing-with-twitter-bootstrap. I'll remove the draggable functionality for now (leave that for future PR), but this might become an issue anyway in specific cases where manually grading questions where jquery-ui is used in a question (e.g., in pl-order-blocks).

File: apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts [hunk 1]
```diff
@@ -278,7 +278,7 @@ export function RubricSettingsModal({ resLocals }: { resLocals: Record<string, a
                             </label>
                           </td>
                           <td>
-                            <div class="form-check">
+                            <div class="form-check form-check-inline">
```
[the remark this answers is not in the document]
Comment (jonatanschroeder) [reply apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts:281]: As mentioned in Slack, the issue was in the template for new items region, which still retained the old behaviour. Fixed in d5cf14369d0834f221d3d93de93a34fe3be0a009.

File: apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts
[the remark this answers is not in the document]
Comment (jonatanschroeder) [reply apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts:1]: Done in 5e7ae0e3349068316632d4b30a4679fe04601b63.

File: apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts [hunk 2]
```diff
@@ -465,6 +265,117 @@ export function RubricSettingsModal({ resLocals }: { resLocals: Record<string, a
   `;
 }
 
+function RubricItemRow(item: RubricData['rubric_items'][0] | null, index: number) {
+  const namePrefix = item ? `rubric_item[cur${item.id}]` : `rubric_item[new]`;
```
Comment (nwalters512) [inline apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts:269]: Thanks for deduplicating all this markup!

I'm assuming there's already JavaScript in place to handle updating these IDs, the order, etc. after instantiating a new row from the template?

**Review by nwalters512 — APPROVED** [review]

File: apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts (hunk 2 above)
Comment (jonatanschroeder) [reply apps/prairielearn/src/pages/instructorAssessmentManualGrading/instanceQuestion/rubricSettingsModal.html.ts:269]: Yes: https://github.com/PrairieLearn/PrairieLearn/blob/aa9182fd912451893e0f4195388e46b7960d4356/apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js#L496-L507

**Pull request merged**